<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsForm</class>
 <widget class="QWidget" name="SettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NTP Server settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <widget class="QComboBox" name="mbtcpComboBox"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>NTP</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Modbus TCP</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>MELSEC</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="melsecComboBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="ntpComboBox"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>GPS</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="gpsComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>IP Address  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ipAddressLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777213</height>
          </size>
         </property>
         <property name="inputMask">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="sendPushButton">
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="receivePushButton">
         <property name="text">
          <string>Receive</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="importPushButton">
         <property name="text">
          <string>Import</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exportPushButton">
         <property name="text">
          <string>Export</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="setTimePushButton">
         <property name="text">
          <string>Set Time</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>General info</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="generalInfoTextEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
